wagtail/admin/templates/wagtailadmin/shared/breadcrumbs.html

```diff
@@ -5,15 +5,16 @@
 
     `pages` - A list of wagtail page objects
     `trailing_breadcrumb_title` (string?) - use this for a non linkable last breadcrumb
+    `classes` - Modifier classes
```

Comment:
I think this variable name needs to be updated.

Reply:
Oops! Thank you 😁 I've updated this now